src/components/ComputeModal.js

```diff
@@ -941,6 +942,9 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
           h(IdContainer, [
             id => div({ style: { gridColumnEnd: 'span 4', marginTop: '0.5rem' } }, [
               label({ htmlFor: id, style: computeStyles.label }, ['Compute type']),
+              h(InfoBox, { style: { marginLeft: '0.5rem', display: requiresStandard() || requiresSpark ? '' : 'none' } }, [
```

Comment:
We tend to pull the predicate out and write it like below
```suggestion
              (requiresStandard() || requiresSpark) && h(InfoBox, { style: { marginLeft: '0.5rem' } }, [
```

This makes the predicate easier to notice when reading the code plus eliminates the need for `? '' : 'none'`.